File: src/main/java/top/axbt/to/service/SelldepositService.java
package top.axbt.to.service;
import java.util.List;

import top.axbt.to.domain.TbSelldeposit;
import top.axbt.to.entity.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SelldepositService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbSelldeposit> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbSelldeposit selldeposit);
	
	
	/**
	 * 修改
	 */
	public void update(TbSelldeposit selldeposit);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbSelldeposit findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbSelldeposit selldeposit, int pageNum, int pageSize);
	
}

File: src/main/webapp/js/service/selldepositService.js
//服务层
app.service('selldepositService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../selldeposit/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../selldeposit/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../selldeposit/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../selldeposit/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../selldeposit/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../selldeposit/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../selldeposit/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});
